### ISSUE TITLE
Changes are not logged when deleting or changing status from the entry list.

### ISSUE THREAD

**Issue by timkelty** [boboldehampsink/auditlog#13]
If you bulk-change statuses (with the checkmarks on the entry list), or delete an entry the same way, the changes are not logged.

**Comment by boboldehampsink** [comment]
Indeed. This is because Craft doesn't fire events on these operations, so I can't hook into it...

**Comment by timkelty** [comment]
Bummer!
Do you know if there is an existing request on http://feedback.craftcms.com/?
I briefly looked but couldn't find anything...could even be considered a bug vs. a feature.

**Comment by timkelty** [comment]
Made one: http://feedback.craftcms.com/forums/285221-feature-requests/suggestions/12350388-raise-events-when-bulk-editing-entry-list-checkbo

**Comment by boboldehampsink** [comment]
Thanks for requesting! I'll keep an eye open on its eventual appearance.

**Comment by carlcs** [comment]
Craft 2.5 fires an onPerformAction event on bulk actions
https://craftcms.com/docs/plugins/events-reference#elements-onPerformAction

**Comment by boboldehampsink** [comment]
Ah didn't know that - interesting!

**Comment by timkelty** [comment]
Great news!

**Comment by fredrik-sogaard** [comment]
@boboldehampsink Any updates on if this is being added to the plugin?
